src/UCTNode.h

```diff
@@ -122,6 +124,10 @@ class UCTNode {
     float m_policy;
     // Original net eval for this node (not children).
     float m_net_eval{0.0f};
+    // Variable used for calculating variance of evaluations.
+    // Initialized to small non-zero value to avoid accidental zero variances
+    // at low visits.
+    std::atomic<float> m_squared_eval_diff{0.01f};
```

Comment:
This seems like a pretty high starting value considering examples of say 2 visits with 51% vs 52% winrates?

Reply:
This isn't that important anymore with minimum visit ratio. I set it to a smaller value, but it could probably be even set to 0 now.